output-multi.cpp

```diff
@@ -364,7 +364,7 @@ int output_multi_t::process_relation(osmid_t id, const memberlist_t &members,
         //all this trickery
         int make_boundary, make_polygon = 1;
         taglist_t outtags;
-        filter = m_tagtransform->filter_rel_member_tags(tags, filtered, m_relation_helper.roles,
```

Comment:
If it was done that way intentionally, then I assume it's so that you don't have to repeat (potentially complex / time-consuming) tag processing and normalisation that you might have already done in the relation filter function again in the relation members filter function. For example, if you had a complex method for figuring out whether a relation was a particular type of feature based on various tags then set `INT_feature=1`, you could at least re-use that in the members filter function without needing to recompute it.

But it's just as likely to be hysterical raisins :wink: